InfoHandler.cpp

```diff
@@ -217,11 +226,16 @@ void InfoHandler::applyInfoToView(SharedAnalysisInfo info, BinaryViewRef bv)
             defineReference(bv, mi.address, mi.typeAddress);
             defineReference(bv, mi.address, mi.implAddress);
 
-            applyMethodType(bv, ci, mi);
+            applyMethodType(bv, ci, methodSelfType, mi);
         }
 
+		for (const auto& ii : ci.ivarList.ivars) {
+			defineVariable(bv, ii.address, ivarType);
+		}
```

Comment:
Remove braces since they are not used with single-line for loops in this project.